client/ayon_core/hosts/nuke/api/lib.py

```diff
@@ -1268,6 +1291,9 @@ def create_write_node(
                 link.setFlag(0x1000)
                 GN.addKnob(link)
 
+    # Adding render farm submission button.
+    add_button_render_farm(GN)
```

Comment:
lets add the button on demand. How about to add it as feature item into `ayon+settings://nuke/create/CreateWriteRender/instance_attributes`. Something like `Instant Farm submission` or something similar?

Reply:
This feature sounds similar to a feature implemented in Houdini by @MustafaJafar where I believe he named it something like "Self-publish" or alike.

Reply:
I think `Headless Farm Submission` would work.